js/ui-routes/main/main.js

```diff
@@ -22,6 +22,15 @@ app.controller('MainController', function($scope, KeyframeFactory, nwguiFactory,
 	$scope.editor = "nothing";
 
 
+	$scope.showFileTab = false;
+	$scope.toggleActive = function() {
+	console.log("scope.showFileTab", $scope.showFileTab);
+	console.log("NG CLICK");
```

Comment:
Will need to revisit this and find a better way to fix the active state on the file info. Clicking the tab to the right is supposed to hide the gray text box containing the branch and commit time info.